go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/99designs/gqlgen v0.17.41
 	github.com/PuerkitoBio/rehttp v1.3.0
 	github.com/aws/aws-sdk-go v1.47.9
-	github.com/aws/aws-sdk-go-v2 v1.22.2
```

Comment:
When I fixed it [here](https://github.com/evergreen-ci/evergreen/pull/7273/files), I also bumped everything to the latest. It seemed to fix everything but it still caused issues for the deploy email. So I'm worried this might still happen again.

Reply:
But it looks like this is bumping it one past it a problematic one, so we'll probably have better luck this time. As long as we keep an eye on the deploy email, that works for me.